hive-plugins/src/main/java/co/cask/plugin/etl/batch/source/HiveSourceConfig.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.plugin.etl.batch.source;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.plugin.etl.batch.HiveConfig;
+
+import javax.annotation.Nullable;
+
+/**
+ * Configurations for Hive batch source
+ */
+public class HiveSourceConfig extends HiveConfig {
+  @Name(Hive.FILTER)
+  @Description("Hive expression filter for scan. This filter must reference only partition columns. " +
```

Comment:
can't tell what this should be from this description.  Is this like the where clause in a query?